config/software/datadog-agent.rb

```diff
@@ -184,6 +184,9 @@
     # The GUI also needs to have the certificate in its folder to send flares
     copy "datadog-cert.pem", "#{install_dir}/dist/datadog-cert.pem"
 
+    copy "#{install_dir}/dist/msvc*.dll", "#{install_dir}/embedded/DLLs"
```

Comment:
could you just add a comment above these 2 lines to explain that we need this for windows server 2008?

Will help get rid of these lines when we drop the support of that version of windows